modules/RTC/TraceablePeerConnection.js

```diff
@@ -2108,6 +2108,67 @@ TraceablePeerConnection.prototype._adjustLocalMediaDirection = function(
     return localDescription;
 };
 
+/**
+ * Munges the stereo flag as well as the opusMaxAverageBitrate in the SDP, based
+ * on values set through config.js, if present.
+ *
+ * @param {RTCSessionDescription} description that needs to be munged.
+ * @returns {RTCSessionDescription} the munged description.
+ */
+TraceablePeerConnection.prototype._mungeOpus = function(description) {
+    const parsedSdp = transform.parse(description.sdp);
+    const mLines = parsedSdp.media;
+
+    for (const mLine of mLines) {
+        if (mLine.type === 'audio') {
+            const { payload } = mLine.rtp.find(protocol => protocol.codec === 'opus');
```

Comment:
Can you please add mimeType for Opus here https://github.com/jitsi/lib-jitsi-meet/blob/master/service/RTC/CodecMimeType.js and use that for comparison ?

Reply:
Done!